conditional.js

```diff
@@ -121,6 +121,19 @@ function generateEvaluateConditionFn(self, ast, $global, _filters, $valueFn, pip
         return values;
       });
     }
+    if (typeof target === "object" && target.split) {
+      let reconstitutedPath = astToCluesPath(target.split);
+      let [ optClause, optSeparator] = reconstitutedPath.match(/(?:,([^),]+))?$/); /* Use the last comma outside parens to determine any optional separator */
+      let path = optClause ? reconstitutedPath.slice(0,-optClause.length) : reconstitutedPath;
+      let separator = optSeparator || ",";
+      separator = separator.replace(/^["'](.+(?=["']$))["']$/, '$1'); // strip quotes
```

Comment:
oh this nonsense should probably be handled in the pegjs, right?

Reply:
I can't get the separator in the pegjs, because it gets gobbled up by the PathList.  I originally had it there, but it never populated.